packages/integration-tests/src/test-projects/conversation-handler/amplify/backend.ts

```diff
@@ -2,12 +2,28 @@ import { defineBackend } from '@aws-amplify/backend';
 import { ConversationHandlerFunction } from '@aws-amplify/ai-constructs/conversation';
 import { auth } from './auth/resource.js';
 import { data } from './data/resource.js';
+import path from 'path';
+import { fileURLToPath } from 'url';
 
 const backend = defineBackend({ auth, data });
 
 const stack = backend.createStack('conversationHandlerStack');
 
-new ConversationHandlerFunction(stack, 'conversationHandlerFunction', {
+new ConversationHandlerFunction(stack, 'defaultConversationHandlerFunction', {
+  models: [
+    {
+      modelId: 'anthropic.claude-3-haiku-20240307-v1:0',
```

Comment:
if we are reusing these IDs lets pull them out

Reply:
we can do that.

Reply:
done.